CHANGES.rst

```diff
@@ -103,6 +103,9 @@ astropy.units
 - Added integrated flux unit conversion to ``spectral_density`` equivalency.
   [#10015]
 
+- Fixed ``u.Quantity`` not taking into account ``ndmin`` if constructed from
+  another ``u.Quantity`` instance with different but convertible unit [#10066]
```

Comment:
Thanks! Change log should be under a bug fix release but I'll let @bsipocz decide whether it should be 4.0.1 or 4.0.2.

I'll also let others to decide if this needs squashing. 😬

Reply:
either would work, I can move around stuff this close to the release

Reply:
I moved it to 4.0.1